src/go/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/systemd/systemd/src/go
+
+go 1.23.5
+
+require github.com/coreos/go-systemd/v22 v22.5.0
+
+require (
+	github.com/foxboron/go-uefi v0.0.0-20241219185318-19dc140271bf // indirect
```

Comment:
Is this a bad rebase maybe? I don't understand why that pulls uefi here.

Reply:
@baloo Ah sorry, I was originally going to rewrite some ASN1 `sbsign` code :)

Will fix tomorrow with CI failures!